goodtables.yml

```diff
@@ -0,0 +1,9 @@
+datapackages:
```

Comment:
https://docs.goodtables.io/getting_started/index.html

`goodtables.yml` is goodtables.io configuration file. It easy to tweak using the documentation.